pkg/stores/partition/store_test.go

```diff
@@ -0,0 +1,1832 @@
+package partition
```

Comment:
Nothing to Change here; just a suggestion for future tests. If you leverage `testify.Suites` It is easier to break up tests into separate functions.